test/e2e_new/multiple_partition_config/install.go

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2021 The Knative Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package multiple_partition_config
```

Comment:
Can we rename this as per Go conventions https://blog.golang.org/package-names?

Reply:
`multiplepartitionconfig` is any better? It sounds just harder to read to me. The reason I used this name is to keep it consistent with `single_partition_config`, if you think this is important I can change both

Reply:
Nope, it's not important for me!

Reply:
and sound good to me!

Reply:
Hopefully we'll get rid of this as soon as this https://github.com/knative-sandbox/reconciler-test/issues/151 is fixed